src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -135,14 +139,15 @@ private void setupListView() {
         setupKeyboardShortcuts();
         setupContextMenu();
 
-        listView.setOnItemSelected(i -> {
+        listView.setOnItemSelected((i, e) -> {
```

Comment:
The `e` parameter can have a better name. `wasRightClick` or something? Then we can do `if (wasRightClick) { ... }` below.